doc/v2.md

```diff
@@ -0,0 +1,20 @@
+## Guide for Migration from v1 to v2
```

Comment:
I'd suggest renaming this file, e.g. `migrating-to-v2.md` or something

Reply:
Done!